mlir/test/Dialect/Rock/integration/reduce/blockwise_reduce/blockwise_reduce_vector_nr_and_scalar_r.mlir

```diff
@@ -29,11 +29,12 @@
 #transform_map23_iter = #rock.transform_map<#map14_iter by [<Unmerge{1, 1, 4, 4} ["m_i", "blk_row", "vec_group", "vec_item"] at [0, 2, 4, 5] -> ["gemmM"] at [0]>, <Unmerge{1, 1} ["n_i", "blk_col"] at [1, 3] -> ["gemmN"] at [1]>] bounds = [1, 1, 1, 1, 4, 4] -> [16, 1]>
 
 func.func @rock_blockwise_vector_nr_and_scalar_r(%input : memref<1x64x384xf32>,  %output : memref<1x64x384xf32>) attributes{arch = "", block_size = 64 : i32, grid_size = 24 : i32, kernel} {
+  %c0 = arith.constant 0 : index
   %input_reg = rock.alloc() : memref<16xf32, #gpu.address_space<private>>
   %output_reg = rock.alloc() : memref<16xf32, #gpu.address_space<private>>
-  %ws_lds = rock.alloc() : memref<64xf32, #gpu.address_space<workgroup>>
```

Comment:
if rock.alloc() are always supposed to allocate i8, should we add that as a check in GpuAllocOp::verify()?

Reply:
It's more that the LDS reduce pass fails if you don't do this so ... yeah, I'll add a check.